client/app/pages/queries/query.html

```diff
@@ -135,8 +135,7 @@ <h3>
                 <span class="zmdi zmdi-refresh"></span> Refresh Schedule</span>
             </td>
             <td class="p-t-15 text-right">
-              <schedule-dialog show="showScheduleForm" schedule="query.schedule" refresh-options="refreshOptions" update-query="updateQueryMetadata" on-close="closeScheduleForm"></schedule-dialog>
-              <schedule-phrase ng-click="openScheduleForm()" is-link="true" schedule="query.schedule" is-new="query.isNew()" />
```

Comment:
👍

Reply:
yeah, this one was really ugly (I clearly understand that it wasn't possible to do it better at that moment)